src/presence_analyzer/static/presence_start_end.html

```diff
@@ -96,33 +94,39 @@
                     if(selected_user) {
                         loading.show();
                         chart_div.hide();
-                        
+                        $.getJSON("/api/v1/presence_start_end/"+selected_user,
+                            function(result) {
+                                final_result = [
+                                ["Mon", new Date(Date.parse("1,1,1,"+result["Monday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Monday"]["end"]))],
+                                ["Tue", new Date(Date.parse("1,1,1,"+result["Tuesday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Tuesday"]["end"]))],
+                                ["Wed", new Date(Date.parse("1,1,1,"+result["Wednesday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Wednesday"]["end"]))],
+                                ["Thu", new Date(Date.parse("1,1,1,"+result["Thursday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Thursday"]["end"]))],
+                                ["Fri", new Date(Date.parse("1,1,1,"+result["Friday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Friday"]["end"]))],
+                                ]
```

Comment:
This is wrong
1) create some helper for processing this data,
2) ur passing from the api all 7 days but using here only 5? So pass 5 and just loop over them here
2) in js use singlequotes,